terraform/ecs_fargate/ecr.tf

```diff
@@ -0,0 +1,8 @@
+/*
+
+# Used to create the ECR but commented out as it's not to be deleted
+
+resource "aws_ecr_repository" "python_ecr_repository" {
+  name = "${var.project_name}-repo"
+}
+*/
```

Comment:
do we need this commented code?

Reply:
Yeah not sure, I guess to have it there in case the ECR is deleted so its easy to re create? Should address it in the README also

Reply:
ok make changes to the README before you merge then

Reply:
done